### PR TITLE
VG-5389: Updates to support list of permissions and saved searches

### PR DESCRIPTION
**Pull request by newfoundgeo**

### CODE REVIEW

**Comment by newfoundgeo** [comment]
@scottlepp Could you review now that I have the tests passing? Thanks!!!

File: app/src/taskrunner/params/list-param.js [hunk 1]
```diff
@@ -1,29 +1,62 @@
 'use strict';
 
 angular.module('taskRunner')
-    .directive('vsListParam', function (authService) {
-
+    .directive('vsListParam', function ($compile, $templateCache, authService, savedSearchService) {
```
Comment (scottlepp) [inline app/src/taskrunner/params/list-param.js]: @newfoundgeo looks like $templateCache isn't used?

File: app/src/taskrunner/params/param-service.js [hunk 2]
```diff
@@ -73,27 +73,27 @@ angular.module('taskRunner').
...
-                    });
-                });
-            }
-
-            if (param.type === 'StringChoice' && param.name !== 'saved_searches') {
-                if (!param.value && angular.isDefined(param.choices)) {
-                    param.value = param.choices[0];
-                }
-            }
+            //if (param.type === 'StringChoice' && param.name === 'saved_searches'){
```
Comment (scottlepp) [inline app/src/taskrunner/params/param-service.js]: @newfoundgeo please remove comments if this code isn't needed

File: test/spec/taskrunner/task-controller-spec.js [hunk 3]
```diff
@@ -52,21 +52,21 @@ describe('TaskCtrl', function () {
...
-        // httpMock.expectJSONP(new RegExp('ssearch')).respond({response: {docs: [{id: 'id', place: '0 0 0 0'}]}});
-        var stateParams = {task:{name:'name'}};
-        controllerService('TaskCtrl', {$scope: scope, $stateParams:stateParams, $state: $state});
-
-        httpMock.flush();
-
-        httpMock.expectJSONP(new RegExp('ssearch')).respond({response: {docs: [{id: 'id', place: '0 0 0 0'}]}});
-        timeout.flush();
-    }
+    //function initCtrl2() {
```
Comment (scottlepp) [inline test/spec/taskrunner/task-controller-spec.js]: @newfoundgeo please remove comments if this code isn't needed.  Did something change where you don't need this?

**Comment by newfoundgeo** [comment]
cleanup the code and comments.

**Pull request merged**